Bulldozer.CSV/SampleCSVs/entity-notes.csv

```diff
@@ -0,0 +1,2 @@
+﻿Id,NoteType,Caption,IsAlert,IsPrivateNote,NotificationSent,Text,EntityTypeName,EntityId,DateTime,CreatedByPersonId
+4183,Group Member Note,,0,0,0,Visited with Sara Meese and asked to join the group right away.,Rock.Model.GroupMember,112_147,2021-03-25 17:22:20.473,
```

Comment:
Probably just nitpicky, but does it matter the date format? Previous csv's https://github.com/KingdomFirst/Bulldozer/blob/master/Bulldozer.CSV/SampleCSVs/person.csv?plain=1 and https://github.com/KingdomFirst/Bulldozer/blob/master/Bulldozer.CSV/SampleCSVs/attendance.csv?plain=1 have American date format.
```suggestion
4183,Group Member Note,,0,0,0,Visited with Sara Meese and asked to join the group right away.,Rock.Model.GroupMember,112_147,03/25/2021 17:22:20.473,
```